src/main/java/seedu/address/logic/commands/meeting/OpenMeetingCommand.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands.meeting;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+import seedu.address.model.meeting.UrlOpener;
+import seedu.address.model.meeting.UrlOpenerManager;
+import seedu.address.model.meeting.exceptions.UnsupportedDesktopException;
+
+/**
+ * Opens a meeting's URL on the device's default browser. <br>
+ * The {@code execute} method of this class creates a {@code UrlOpenerManager} which checks the running environment
+ * for desktop functionality support and passes it to {@code executeWithUrlOpener}.
+ */
+public class OpenMeetingCommand extends Command {
+    public static final String COMMAND_WORD = "open";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens the meeting identified by the index number used in the "
```

Comment:
I think this can be updated to:

```suggestion
            + ": Opens the meeting URL identified by the index number used in the "
```

Reply:
I think `meeting URL identified by the index number` has a different meaning than the URL of the meeting identified by the index number. How about `Opens the URL of the meeting identified by the index number`?